### ISSUE TITLE
关于conv layer的Batch Normalization

### ISSUE THREAD

**Issue by ylqi007** [YunYang1994/tensorflow-yolov3#382]
![image](https://user-images.githubusercontent.com/23485976/68075478-4cebc980-fd76-11e9-9c6a-204d2a36cc47.png)

@YunYang1994 
你好，大神。在代码中，关于`convolutional`的定义有一处疑惑，如果有空的话，还请指教一下：

在做BN处理的时候,只对conv做了`batch_normalization`,然后就直接进行`activate`处理,并没有+bias. 这是故意这样做的吗?

谢谢!

**Comment by YunYang1994** [comment]
是的

**Comment by ylqi007** [comment]
能解释下为什么吗？

**Comment by jiarenyf** [comment]
因为：conv+bias+bn=conv+bn...

**Comment by YunYang1994** [comment]
这样做是为了减少参数，使得网络更加轻量级。你仔细去看很多backbone，比如resnet的源码就会发现，一般都是在每个block的最后面layer设置use_bias=True，而在中间layer设置use_bias=False。偏置bias的作用有多大，为什么要有这个？你得复习下神经网络的基础知识。
